src/emulator/cpu.ts

```diff
@@ -411,6 +505,15 @@ export class Cpu {
                 return instruction.cycles;
             }
 
+            case Operation.stop:
```

Comment:
I would leave that as `NOP` or as an undefined opcode for now. The real behaviour is very quirky, and I am reasonably confident that few or no games use it.

Reply:
since nothing was done here other than setting a boolean, we could also leave it in. But to make it clear that it is not further implemented, we can make an output with system.error or system.warn here